src/modules/tests/test-data/common/controlled-stop/controlled-stop.reducer.ts

```diff
@@ -1,13 +1,20 @@
+import { CatBUniqueTypes } from '@dvsa/mes-test-schema/categories/B';
 import { CatFUniqueTypes } from '@dvsa/mes-test-schema/categories/F';
+
 import * as controlledStopActions from './controlled-stop.actions';
 import { CompetencyOutcome } from '../../../../../shared/models/competency-outcome';
 
 export const initialState: CatFUniqueTypes.ControlledStop = {};
 
+// TODO: Update schemas to create common controlled stop
+export type ControlledStopUnion =
+  | CatBUniqueTypes.ControlledStop
+  | CatFUniqueTypes.ControlledStop;
```

Comment:
to be correct shouldn't this also union the other home tests

Reply:
It should yes, will add them.